### DISCUSSION TITLE
Changing the background in the Developer Portal

### DISCUSSION THREAD

**Discussion by marius-terciu** [Azure/api-management-developer-portal#2091] [General]
I'm new to this portal and I'm trying to customize it. I've watched the [video](https://www.youtube.com/watch?v=5mMtUSmfUlw) from MS Docs where it shows how to do it, but it seems I cannot select the whole section to edit it (I can select the whole header, the menu, the logo, and the artichle). Is something I'm doing wrong ?
PS. I'm using the managed portal, not the self hosted one and I've tried both with Edge and Mozilla Firefox and I cannot see the edit button for the whole section.

**Comment by Piekop** [comment]
The lack of support is huge on this developer portal

**Comment by malincrist** [comment]
Hello @marius-terciu, thanks for reaching out to us.
When you select the article, you can click on the button from the left of "edit article", with the 2 arrows. This button will select the parent component, which is the section:
![edit-section](https://user-images.githubusercontent.com/92857141/218994177-3de42542-c171-4efa-8aed-3fefc8932a89.gif)

Hope this helps! :)

**Reply by marius-terciu** [reply]
@malincrist Thanks a lot! 👍 It works
